src/app/dashboard/revalidation-notes/revalidation-notes.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit } from "@angular/core";
+import { MatBottomSheetRef } from "@angular/material/bottom-sheet";
+
+@Component({
+  selector: "app-revalidation-notes",
+  templateUrl: "./revalidation-notes.component.html",
+  styleUrls: ["./revalidation-notes.component.scss"]
+})
+export class RevalidationNotesComponent implements OnInit {
+  messages: string[]; // mock
+  addEditNote: boolean;
```

Comment:
same as below, this state should ideally live inside ngxs state management

Reply:
agreed, and also table datasource etc. all hard coded for now